dag_files/dag_walkthrough.yml

```diff
@@ -13,3 +13,9 @@ steps:
   - data://garden/un/2022-07-11/un_wpp
   data://grapher/who/2022-07-17/who_vaccination:
   - data://garden/who/2022-07-17/who_vaccination
+  data://meadow/who/2022-09-30/ghe:
+  - walden://who/2022-09-30/ghe
+  data://garden/who/2022-09-30/ghe:
+  - data://meadow/who/2022-09-30/ghe
+  data://grapher/who/2022-09-30/ghe:
+  - data://garden/who/2022-09-30/ghe
```

Comment:
Here I suppose the `data://garden/reference/`dataset should be included as a dependency of the garden step, since the country_code_to_country function uses it.